features/admin/sign_in.feature

```diff
@@ -1,8 +1,8 @@
 @administrator_login
-Feature: Sign in to the store
-    In order to view my orders
-    As a Visitor
-    I want to be able to log in to the store
+Feature: Sign in to the administration panel
+    In order to access administration panel
```

Comment:
We should avoid descriptions `In order to X... I want to X` 💃 Accessing the administrator panel is a way to fulfil some goal. Maybe sth like `In order to be able to manage my store`?

Reply:
You should update contributing documentation then, because everywhere you mention current version as 1.7 https://docs.sylius.com/en/latest/book/contributing/code/patches.html#choose-the-right-base-branch
Should I open a new pull request to 1.8 now?

If we are talking about feature description I just tried to follow your standards, because the old description didn't describe the sign in to the administration panel, but was a copy of the sign in to the customer panel (https://github.com/Sylius/Sylius/blob/master/features/account/signing_in.feature)